Closure_Front_End/src/components/PieChart.vue

```diff
@@ -0,0 +1,22 @@
+
+<script>
+import { defineComponent } from 'vue';
+import { Pie } from 'vue3-chart-v2';
+
+export default defineComponent({
+  name: 'MonthlyChart',
+  extends: Pie,
+  mounted () {
+    // Overwriting base render method with actual data.
+    this.renderChart({
+      labels: ['Must', 'Choose from list', 'Choice'],
+      datasets: [
+        {
+          backgroundColor: ['#bc87d0','#fbaf5d','#f06eaa'],
+          data: [70, 20, 40]
```

Comment:
What do these values mean?

Reply:
just an approximate distribution of your points on a standard single major degree, it's just a dummy graph

Reply:
Why a dummy?

Reply:
because untill we find a way communicating the data between different views, i can't access them from the analytics view. thats an architectural decision that noa should do i believe

Reply:
1. Is this feature as part of this PR necessary for creating the infrastructure?
2. Can we hide it until it's ready? (so `master` doesn't have half-baked features)